autoware_map_msgs/srv/LoadPCDMaps.srv

```diff
@@ -0,0 +1,22 @@
+# Mode definition
+uint8 MODE_PARTIAL_AREA_LOAD=0
+uint8 MODE_DIFFERENTIAL_AREA_LOAD=1
+
+# Query mode
+uint8 mode
+
+# Area query for map loading
+# (PCD grids that overlaps with the area is going to be loaded)
+AreaInfo area
+
+# The IDs of PCD maps that the node already has
+# (only used when mode = MODE_DIFFERENTIAL_AREA_LOAD)
+string[] already_loaded_ids
+
+---
+# Newly loaded PCD maps with ID
+PCDMapWithID[] loaded_pcds
+
+# IDs of map that satisfies the query and is already loaded
+# (only used when mode = MODE_DIFFERENTIAL_AREA_LOAD)
+string[] already_loaded_ids
```

Comment:
already_loaded_ids means "should be removed map ids"?

Reply:
No, it literally means "the map ids that is already loaded", or better to state "the map ids that satisfies the query and is already loaded". Mathematical expression [here](https://github.com/kminoda/autoware_msgs/blob/feat/add_autoware_map_msgs_for_dynamic_map_loading/autoware_map_msgs/README.md#mode-2-differential-map-loading) may help the understandings.

But I think it is also valid to send "ids that should be removed". Do you have any preference on these two?

Reply:
I changed the response to "ids that should be removed" at this commit https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/d3706f92918b734cf1736f93222d03ddb26bcb46

Reply:
LGTM :+1: